content/asset-tracking/using-the-sdks.textile

```diff
@@ -64,79 +64,91 @@ You can find information on installing the Ably Asset Tracking SDKs in the follo
+      .connection(ConnectionConfiguration(Authentication.tokenRequest(CLIENT_ID) { requestParameters ->
+          // get TokenRequest from your server
+          getTokenRequestFromAuthServer(requestParameters); // customer implements this function
+          }))
+  ```
 
-```[javascript]
-/* authURL is the endpoint for your authentication server. It returns either
-   a `TokenRequest` or a `Token` */
-const subscriber = new Subscriber({ 
-  authUrl: 'http://my.website/auth',
-  clientId: 'CLIENT_ID'
-})
-```
+  ```[javascript]
+  /* authURL is the endpoint for your authentication server. It returns either
+    a `TokenRequest` or a `Token` */
+  const subscriber = new Subscriber({ 
+    authUrl: 'http://my.website/auth',
+    clientId: 'CLIENT_ID'
+  })
+  ```
 
-You can read more about token authentication in our "documentation":/core-features/authentication#token-authentication.
+  You can read more about token authentication in our "documentation":/core-features/authentication#token-authentication.
 
-h3(#jwt-authentication). JWT Authentication
+  h3(#jwt-authentication). JWT Authentication
 
-The following example demonstrates establishing a connection using JWT authentication:
+  The following example demonstrates establishing a connection using JWT authentication:
 
-```[android]
-val publisher = Publisher.publishers() // get the Publisher builder in default state
+  ```[android]
+  val publisher = Publisher.publishers() // get the Publisher builder in default state
     .connection(ConnectionConfiguration(Authentication.jwt(CLIENT_ID) { tokenParameters ->
-        // get JWT from your server
-        getJWTFromAuthServer(tokenParameters); // customer implements this function
-        }))
-```
+          // get JWT from your server
+          getJWTFromAuthServer(tokenParameters); // customer implements this function
+          }))
+  ```
 
-```[javascript]
-// authURL is the endpoint for your authentication server. It returns a JWT
-const subscriber = new Subscriber({ 
-  authUrl: 'http://my.website/auth',
-  clientId: 'CLIENT_ID'
-})
-```
+  ```[javascript]
+  // authURL is the endpoint for your authentication server. It returns a JWT
+  const subscriber = new Subscriber({ 
+    authUrl: 'http://my.website/auth',
+    clientId: 'CLIENT_ID'
+  })
+  ```
 
-You can read more about JWT authentication in our "documentation":/core-features/authentication#ably-jwt-process.
+  You can read more about JWT authentication in our "documentation":/core-features/authentication#ably-jwt-process.
 
-blang[android].
+blang[swift].
+  The client requires authentication in order to establish a connection with Ably. You authenticate with your Ably API key (available in "your account dashboard":https://ably.com/accounts) and can optionally "identify the client with a client ID":/realtime/authentication#identified-clients. The following example demonstrates how to achieve this:
```

Comment:
Is it a good idea to add some mention of Token auth not being available on iOS, it will be released soon? And a note to update this documentation when that does feature does get released soon.

Reply:
I agree that a note mentioning that you can only use basic auth for iOS would be useful to highlight the difference from Android. No mention of the "coming soon" though :-)

Reply:
Done, with no mention of "coming soon" 😄